app/src/main/java/com/compscieddy/meetinthemiddle/GroupActivity.java

```diff
@@ -498,6 +500,39 @@ public void onMapClick(LatLng point) {
   }
 
   @Override
+  public void onMapDrag() {
+    if (!collapsed){
+      Display display = getWindowManager().getDefaultDisplay();
+      Point size = new Point();
+      display.getSize(size);
+      int height = size.y;
+
+      ResizeAnimation resizeAnimation;
+
```

Comment:
@ambardhesi but there's also this code below - move it out into a method so both places just call that method